rust/agama-cli/src/logs.rs

```diff
@@ -45,6 +51,42 @@ pub async fn run(subcommand: LogsCommands) -> anyhow::Result<()> {
     }
 }
 
+// Whatewer passed in dest formed into an absolute path with archive name
+// if dest is none then a default is returned
+// if dest is directory then a default file name for the archive will be appended
+// if dest is path with a file name then it is used as is a name for resulting archive
+fn parse_dest(dest: Option<PathBuf>) -> Result<PathBuf, io::Error> {
```

Comment:
You do not need to handle an `Option` if you set a default value for the `dest` field of `LogsCommands::Store`. But it is fine as it is.

Reply:
i did it this way because it was for free and kept the dispatch function a little bit simpler.